src/v0/destinations/klaviyo/transform.js

```diff
@@ -107,24 +140,25 @@ const identifyRequestHandler = async (message, category, destination) => {
 // ----------------------
 
 const trackRequestHandler = (message, category, destination) => {
-  let payload = {};
-  if (!destination.Config.publicApiKey) {
-    throw new ConfigurationError('Public API Key is a required field for track events');
-  }
+  const payload = {};
   let event = get(message, 'event');
   event = event ? event.trim().toLowerCase() : event;
```

Comment:
Do we want to lowercase `event` from message ? As sometimes, we don't want to do it in other destinations right ?! Just let's once think through this one

Reply:
We are doing it after extracting the event below.

Reply:
Ok